Concurrent Modification Exception im DetailAnalysisService	Wenn man mehrere gleiche FetchBatchJobs triggert, kommt es zu einer concurrent modification exception. Konkret passiert das beim hochzählen des tryCounts der FetchJobs. Dabei verabschieden sich die Worker und der Service ist nicht mehr funktionstüchtig.		Bug	OpenSpeedMonitor	17/Nov/16 10:39 AM	17/Nov/16 3:27 PM														17/Nov/16 3:27 PM;msk;Das Problem lag im Worker, der die Queue füllt. Er hat zwar nach Duplikaten gesucht und diese ggfs. gelöscht, dann aber die gelöschten FetchJobs trotzdem gescheduled. Wenn dann viel später der Job abgearbeitet wurde und sein TryCount hohchgezählt werden sollte, kam es zur Exception, weil der FetchJob schon gelöscht war.																																																						
